packages/fxa-graphql-api/src/gql/account.resolver.ts

```diff
@@ -509,6 +516,34 @@ export class AccountResolver {
     };
   }
 
+  @Query((returns) => AccountStatusPayload, {
+    description:
+      'Check the status of an account using session token or uid. ' +
+      'This query is equivalent to the GET /account/status endpoint in auth-server.',
+  })
+  @CatchGatewayError
+  public async accountStatus(
+    @Args('input', { type: () => AccountStatusInput })
+    input: AccountStatusInput
+  ): Promise<AccountStatusPayload> {
+    if (input.token) {
```

Comment:
In auth-server the `/account/status` route is considered optional, therefore I couldn't use the `@GqlSessionToken` decorator. This route can accept a sessionToken id or a uid and check to see if that account exists.